modules/embedding_module.py

```diff
@@ -117,7 +123,7 @@ def compute_embedding(self, memory, source_nodes, timestamps, n_layers, n_neighb
       neighbors = neighbors.flatten()
       neighbor_embeddings = self.compute_embedding(memory,
                                                    neighbors,
-                                                   np.repeat(timestamps, n_neighbors),
+                                                   edge_times.flatten(),
```

Comment:
We actually want to compute the embedding for the neighbors at the current timestamp t, not at the timestamp when the central node had the interaction with the neighbor. This provides more up-to-date information. See Eq. 9 in the paper: https://arxiv.org/pdf/2006.10637.pdf

Reply:
Thanks for your feedback and I have restored this part to the original implementation.